### PR TITLE
fix jump to new conversation of kbfs chats

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers this fixes the jump to new convo flow

### CODE REVIEW

**Comment by chrisnojima** [comment]
I renamed the payload param and flow didn't catch it because the payload types weren't exact. i don't remember why we undid that (it used to be) but it really should go back i think

File: shared/actions/config.js [hunk 1]
```diff
@@ -190,7 +190,7 @@ const getBootstrapStatus = (): AsyncAction => dispatch =>
   new Promise((resolve, reject) => {
     RPCTypes.configGetBootstrapStatusRpcPromise()
       .then(bootstrapStatus => {
-        dispatch(ConfigGen.createBootstrapStatusLoaded(bootstrapStatus))
+        dispatch(ConfigGen.createBootstrapStatusLoaded({...bootstrapStatus}))
```
Comment (chrisnojima) [inline shared/actions/config.js]: wants a copy so its readonly

File: shared/actions/fs-platform-specific.desktop.js [hunk 2]
```diff
@@ -137,6 +137,7 @@ export const installKBFS = () => Saga.call(RPCTypes.installInstallKBFSRpcPromise
 export const installKBFSSuccess = (result: RPCTypes.InstallResult) =>
   Saga.sequentially([
     Saga.call(waitForMount, 0),
+    // $FlowFixMe @jzila please fix
```
Comment (chrisnojima) [inline shared/actions/fs-platform-specific.desktop.js]: @jzila updated flow typing reveals this is incorrect

File: shared/chat/conversation/container.js [hunk 3]
```diff
@@ -69,11 +72,13 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
       type = 'normal'
     }
   } else if (stateProps._pendingConversationUsers) {
-    conversationIDKey = Constants.getExistingConversationWithUsers(
-      stateProps._pendingConversationUsers,
-      stateProps._you,
-      stateProps._metaMap
-    )
+    conversationIDKey = stateProps._findExisting
```
Comment (chrisnojima) [inline shared/chat/conversation/container.js]: the actual fix for the original bug

File: shared/chat/conversation/messages/reset-user/container.js [hunk 4]
```diff
@@ -15,16 +15,16 @@ const mapStateToProps = (state: TypedState, {conversationIDKey}): * => {
 }
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
-  _chatWithoutThem: (username: string, conversationIDKey: Types.ConversationIDKey) =>
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/reset-user/container.js]: we actually remove all pending users in the saga itself so we don't need this param

File: shared/fs/banner/container.js [hunk 5]
```diff
@@ -23,6 +23,7 @@ const mapDispatchToProps = (dispatch: Dispatch) => {
   const uninstall = () => dispatch(FsGen.createUninstallKBFS())
   return {
     getFuseStatus: () => dispatch(FsGen.createFuseStatus()),
+    // $FlowFixMe @jzila please fix
```
Comment (chrisnojima) [inline shared/fs/banner/container.js]: @jzila updated flow typing reveals this is incorrect

File: shared/actions/fs-platform-specific.desktop.js (hunk 2 above)
Comment (jzila) [reply shared/actions/fs-platform-specific.desktop.js]: Good catch, I'll fix in my refactor PR #11149

**Pull request merged**